### ISSUE TITLE
Account login is case sensitive

### ISSUE THREAD

**Issue by Ch0wW** [chrisvel/wreeto_official#49]
Following the default account in README.MD, trying to log in as "**A**dmin" doesn't work, as it doesn't know the account. However, "admin" does work. Is that normal behaviour to be case-sensitive?

**Comment by chrisvel** [comment]
Yes, this is normal but it would be better/less confusing to remove this constraint. I will make the appropriate changes.
